ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WatchedImages/WatchedImagesTable.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { CSSProperties } from 'react';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import { Bullseye, Button } from '@patternfly/react-core';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+
+import { WatchedImage } from 'types/image.proto';
+import { UseRestMutationReturn } from 'hooks/useRestMutation';
+import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
+import { Empty } from 'services/types';
+
+export type WatchedImagesTableProps = {
+    className?: string;
+    style?: CSSProperties;
+    watchedImages: WatchedImage[];
+    unwatchImage: UseRestMutationReturn<string, Empty>['mutate'];
+    isUnwatchInProgress: boolean;
+    'aria-labelledby'?: string;
+};
+
+function WatchedImagesTable({
+    className,
+    style,
+    watchedImages,
+    unwatchImage,
+    isUnwatchInProgress,
+    ...props
+}: WatchedImagesTableProps) {
+    return (
+        <div className={className} style={style}>
+            {watchedImages.length === 0 && (
+                <Bullseye>
+                    <EmptyStateTemplate title="No watched images found" headingLevel="h2" />
+                </Bullseye>
+            )}
+            {watchedImages.length > 0 && (
+                <TableComposable
+                    aria-labelledby={props['aria-labelledby']}
+                    variant="compact"
+                    style={
+                        {
+                            '--pf-c-table--m-compact--cell--first-last-child--PaddingLeft': '0',
+                        } as CSSProperties
+                    }
+                >
+                    <Thead noWrap>
+                        <Tr>
+                            <Th>Image</Th>
+                            <Th aria-label="Remove watched image" />
```

Comment:
Why does <Th> work here, but other places we have to use <Td> when there is no actual content?

Reply:
This works since the `aria-label` gives the table header an accessible name that can be used by screen readers. I think I would only recommend using `<Td>` in the header when there isn't a valid accessible name for the header. Even though it fixes our aXe test issues, I imagine it doesn't provide useful text to the user either. (Would have to validate with an actual screen reader though...)